contributing/tools.md

```diff
@@ -0,0 +1,46 @@
+# Tools that may save you and the team some time
+
+## Automatically format your Git commit with clang-format
+
+This tool allows you to run clang-format on your local changes before they're sent out for review.
+This will minimize the amount of style feedback you get from the team and saves everyone time.
+
+```bash
```

Comment:
I think we’re going to move away from the automated installer because there are enough cases where folks have already installed their own dependencies and our scripts fail to consider these edge cases.